tools/accuracy/accuracy_checker.py

```diff
@@ -116,6 +116,7 @@ def main():
 
     model = migraphx.parse_onnx(model_name, default_dim_value=batch)
 
+    print(model)
```

Comment:
maybe only print this if verbose flag is passed?
`if args.verbose: ...`

Reply:
done